_posts/GaKay/2016-04-06-gao14g-dictionary-exercises.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+author: gao14g
+title: "Ga Kay's MOAR Dictionary Exercises"
+---
+
+Exercise 1:
+<iframe src="https://trinket.io/embed/python3/d2a4bf04f5" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2-5:
+<iframe src="https://trinket.io/embed/python3/a2178ed680" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
This trinket doesnt seem to have your code in it.  Did you get the embed code via the share link?